### PR TITLE
ci(Travis): Dont build branches except master, only build pull requests

### PR DESCRIPTION
**Pull request by JCB-K**

### CODE REVIEW

**Comment by JCB-K** [comment]
@iamstarkov as you can see Travis is still building the PR. But no unnecessary duplicate builds like before.

File: .travis.yml [hunk 1]
```diff
@@ -6,8 +6,8 @@ node_js:
 
 # blacklist version commits
```
Comment (bstream) [inline .travis.yml]: Change or remove this maybe? 🤔

**Comment by bstream** [comment]
Maybe it is also time to drop Node 4 builds in travis. wdyt @iamstarkov @JCB-K ?

File: .travis.yml (hunk 1 above)
Comment (JCB-K) [reply .travis.yml]: @bstream whoops, fixed

**Pull request merged**